repsetdata.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+set -euo pipefail
+
+OUTPUT_NAME="variables-by-replication-set-$(date +"%Y_%m_%d_%I_%M_%s").csv" 
+SELECT_STATEMENT=$(cat <<EOF
+        SELECT  
+          RTRIM(LTRIM(REPLACE(CAST(replication_sets AS varchar), '{}', 'Full' ), '{'),'}') AS "ReplicationSets", 
+          policy_id AS "Policy",
+          identifier(resource_id) as "Variable",
```

Comment:
```suggestion
          identifier(resource_id) as "Variable"
```

The trailing comma results in a syntax error